5.3_hidden_strings.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+
+def read_in_chunks(f, chunk_size):
+    while data := f.read(chunk_size):
+        yield data
+
+
+pattern = rb'[a-z]{5,}!'  # pattern for lowercase letters followed by an exclamation mark
+
+with open('logo.jpg', 'rb') as f:  # open the file in binary
```

Comment:
better to save the file as a relative path, and also put it in a constant and not like that